recipes/ncbi-vdb/meta.yaml

```diff
@@ -41,3 +42,23 @@ test:
     - test -e ${PREFIX}/lib64/libncbi-vdb.so # [not osx]
     - test -e ${PREFIX}/lib64/libncbi-vdb.dylib # [osx]
 
+outputs:
+  - name: ncbi-vdb-py
+
+    build:
+      noarch: python
+      script: |
+        PYVER=`python -c 'import sys; print(str(sys.version_info[0])+"."+str(sys.version_info[1]))'`
+        mkdir -p ${PREFIX}/lib/python${PYVER}/site-packages/
+        cp ./py_vdb/vdb.py ${PREFIX}/lib/python${PYVER}/site-packages/
+
+    requirements:
+      run:
+        - ncbi-vdb ={{ version }}
+        - python >=3.5
+
+    test:
+      imports:
+        - vdb
+      commands:
+        - python -c "import vdb; vdb.manager()"
```

Comment:
I tried this to test if the module is working, but `python` here is the system python, not the conda python. What is the proper way for calling the right Python?

Reply:
(see failed run in https://app.circleci.com/pipelines/github/bioconda/bioconda-recipes/33633/workflows/6328262e-54c5-48e5-a3b9-5d1c70e716f0/jobs/125609)